lib/Fregot/Builtins/Internal.hs

```diff
@@ -159,6 +160,9 @@ instance (Eq a, FromVal a, Hashable a) => FromVal (HS.HashSet a)  where
 -- sort of collection.
 newtype Collection a = Collection [a]
 
+-- | Like `Collection`, but collects an object's keys, not values.
+newtype Collection' a = Collection' [a]
```

Comment:
Open to naming suggestions if we want to keep this.

Reply:
I would call this `Keys` (and optionally we can rename `Collection` to `Values`?)